### DISCUSSION TITLE
Utilisation des structures d'enveloppes directement comme un "folder"?

### DISCUSSION THREAD

**Discussion by nmathey** [MadeInPierre/finalynx#67] [Q&A] [answered]
Hello,

Est-il déjà possible de récupérer directement la structure de certaines enveloppes récupérées ("Account") via l'API pour les utiliser telle quelles comme un "folder" sous FinaLynx?

Ce serait très utile pour les comptes titres avec beaucoup de lignes, les portefeuille crypto avec plusieurs crypto différentes et surtout pour les assurances vie pilotée où les produits peuvent être amener à changer de manière régulière selon les arbitrages.

Exemple pour mon assurance vie WeSave (gestion pilotée), où je souhaiterais créer un "Folder" répliquant directement les lignes du compte "Accroissement de capital" sans devoir le reconstruire ligne par ligne:
<img width="535" alt="Capture d’écran 2023-05-01 à 11 18 33" src="https://user-images.githubusercontent.com/20896232/235383063-48a10dcc-dd5c-41fb-b6a4-badae7a4558b.png">

**Comment by MadeInPierre** [comment]
Malheureusement ce n'est pas encore possible, il faut entrer chaque ligne pour l'instant...

Mais c'est une priorité qui arrive dans les prochains jours (heures?) ! Ce souci est également lié à 2 autres discussions : 
- #47 
- #57 

Je prévois de refaire le système de fetch comme mentionné à la fin de #57, j'incluerai la création automatique de Folders en fonction de filtres sur l'enveloppe en même temps !

**Reply by MadeInPierre** [reply, marked as answer]
Implémenté dans #70 donc je ferme ce sujet !
